test/queued_jobs/queued_jobs_test.go

```diff
@@ -45,16 +61,23 @@ func TestEnqueue(t *testing.T) {
 	test.Assert(t, diff < 100*time.Millisecond, "")
 }
 
-func TestEnqueueNoData(t *testing.T) {
-	test.SetUp(t)
-	defer test.TearDown(t)
-	_, err := jobs.Create(sampleJob)
+func testEnqueueNoData(t *testing.T) {
+	t.Parallel()
+	id, _ := types.GenerateUUID("jobname_")
+	j := models.Job{
+		Name:             id.String(),
+		DeliveryStrategy: models.StrategyAtLeastOnce,
+		Attempts:         7,
+		Concurrency:      1,
+	}
+	_, err := jobs.Create(j)
 	test.AssertNotError(t, err, "")
 
 	expiresAt := types.NullTime{Valid: false}
 	runAfter := time.Now().UTC()
 
-	_, err = queued_jobs.Enqueue(factory.JobId, "echo", runAfter, expiresAt, []byte{})
+	qjid, _ := types.GenerateUUID("job_")
+	_, err = queued_jobs.Enqueue(qjid, j.Name, runAfter, expiresAt, []byte{})
```

Comment:
What is the blank identifier doing?

Reply:
It's a compile error to create a variable that never gets used, so the blank identifier tells the compiler you don't want or need the value present there.

Reply:
More here: https://golang.org/doc/effective_go.html#blank_assign